packages/types/lib/api.ts

```diff
@@ -76,3 +76,8 @@ export interface Endpoint<T extends EndpointDefinition> {
      */
     Reply: ResDefaultErrors | (T['Error'] extends ApiError<any> ? T['Error'] | T['Success'] : T['Success']);
 }
+
+export interface ErrorPayload {
+    type: string;
+    description: string;
+}
```

Comment:
Is this an api related type?

Reply:
internal API I would say

Reply:
if we can keep this file for http api, maybe rename the file if clearer 👌🏻